### PR TITLE
Adapt ansible-artimed to last version of metavisitor tools and workflows

### PR DESCRIPTION
**Pull request by drosofff**

### CODE REVIEW

File: extra-files/metavisitor/metavisitor_tool_list.yml [hunk 1]
```diff
@@ -1,102 +1,244 @@
...
   owner: drosofff
+  revisions:
+  - dbac07fc9186
+  - ac7d8e55bb67
+  - 68f58363f1c6
+  - e4874d1ae69d
+  tool_panel_section_id: metavisitor
   tool_panel_section_label: Metavisitor
-  tool_shed_url: https://toolshed.g2.bx.psu.edu/
+  tool_shed_url: https://testtoolshed.g2.bx.psu.edu/
```
Comment (mvdbeek) [inline extra-files/metavisitor/metavisitor_tool_list.yml]: Is that necessary? If at all possible we should use the main toolshed ones.

Comment (drosofff) [inline extra-files/metavisitor/metavisitor_tool_list.yml]: It's a bug. I'll fix it

**Comment by mvdbeek** [comment]
This looks very nice, but you have to remove the `tool_panel_section_id` from the tool list, see [here](https://github.com/galaxyproject/ansible-galaxy-tools/blob/master/files/tool_list.yaml.sample) for details.

**Comment by drosofff** [comment]
so this part is misleading ?

- name: bedtools
owner: iuc
tool_panel_section_label: 'BED tools'
revisions:
- 2cd7e321d259

Christophe Antoniewski

ARTbio <http://artbio.fr/> - Tel +33 1 44 2*7 70 05*
Drosophila Genetics & Epigenetics <http://drosophile.org/> - Tel +33 1 44 2
*7 34 39*Mobile +33 6 68 60 51 50
https://twitter.com/ARTbio_IBPS
https://twitter.com/drosofff

2016-03-18 19:27 GMT+01:00 Marius van den Beek <notifications@github.com>:

**Comment by mvdbeek** [comment]
No, this is OK, but if you specify both `tool_panel_section_id` and `tool_panel_section_label`, the id is being preferred, and since this doesn't exist on new instances, the tools either won't install or will install outside of the indicated tool panel section.

File: extra-files/metavisitor/metavisitor_tool_list.yml [hunk 2]
```diff
@@ -1,102 +1,244 @@
...
 - name: ncbi_blast_plus
   owner: devteam
   revisions:
+  - 623f727cdff1
+  - 577d9c12411a
   - 7f3c448e119b
-  tool_panel_section_label: Blast plus suite
+  tool_panel_section_id: ncbi_blast+
```
Comment (mvdbeek) [inline extra-files/metavisitor/metavisitor_tool_list.yml]: if you remove all occurrences of `tool_panel_section_id` you get what you expect.

**Comment by drosofff** [comment]
Comment n°1
There an issue with the Travis test, because I have now tested this PR on an IFB instance and it works.

**Comment by mvdbeek** [comment]
The build is timing out during the upload to the dockerhub, but you can see that the remaining test execute fine. I'll increase the timeout.

**Comment by drosofff** [comment]
Comment n°2, still in the context of multiple tests in IFB instances:

Ansible exits with 0 errors, but there is a number of package that have to be repaired by hands. (perl, python, cairo, etc). It seems to correlate with need for download of large file in galaxy package repo, for instance:
```
Attempting to download via url: http://depot.galaxyproject.org/package/source/cairo/cairo-1.12.14.tar.bz2
```
Just a guess

**Comment by mvdbeek** [comment]
Yes, that was supposed to be fixed with https://github.com/galaxyproject/ansible-galaxy-tools/pull/17,
I'll push an update to the role.

**Comment by drosofff** [comment]
And anyway, the package_python_2_7 resists to any repair/reinstallation (readline is the suspect, or preexisting entry in the database that preclude update)

**Comment by mvdbeek** [comment]
Well, the role didn't have https://github.com/galaxyproject/ansible-galaxy-tools/pull/17 included, so that means ansible caused galaxy to restart during the tool installation, because it was assuming that the tool installation was finished. That's hard to recover from.

**Comment by mvdbeek** [comment]
Alright, this will not affect the tests anyway, for me it's OK. We should probably remove the outdated revisions, but then can be the next PR.

**Comment by drosofff** [comment]
Thanks, anyway there is still work to be done on the workflows that still use outdated tools and there compatibility with the ansible/bioblend installed tools.

**Pull request merged**